.github/workflows/check.yaml

```diff
@@ -32,12 +32,12 @@ jobs:
     with:
       enable-bioccheck: true
   coverage:
-    name: Coverage 📔 
+    name: Coverage 📔
```

Comment:
What with `if` condition?

Reply:
As discussed, let's enable coverage check also for pushes to `main`.